.github/actions/write-file/action.yml

```diff
@@ -66,12 +67,17 @@ runs:
           exit 1
         fi
 
+        echo "# Pushing changes"
         cd "${{ inputs.checkout_path }}"
+        git status
```

Comment:
nitpick: is it useful to keep this for the build logs?

Reply:
It's not required but it's useful IMO, it's just logging.

It's convenient for debugging but also for reference.